pkg/providers/amifamily/bootstrap/eksbootstrap.go

```diff
@@ -119,6 +119,14 @@ func (e EKS) isIPv6() bool {
 	if e.KubeletConfig == nil || len(e.KubeletConfig.ClusterDNS) == 0 {
 		return false
 	}
+
+	ips := strings.Split(e.KubeletConfig.ClusterDNS[0], ",")
```

Comment:
ClusterDNS is already a string slice, can we just use the slice directly? All of these AMIFamilies should have support now for passing all of the DNS entries through

Reply:
Ok. you mean we just respect all the clusterDNS strings and convert as a string comma-separated string with all IPs

Reply:
Right. Ideally we don't let you do a comma-delimited list in any of the strings for ClusterDNS and you just specify them as separate strings in the structued object.